scripts/juju_inventory_script.py

```diff
@@ -32,7 +35,8 @@ def main():
                     'username': os.getenv('JUJU_USERNAME'),
                     'password': os.getenv('JUJU_PASSWORD'),
                     'model_uuid': os.getenv('JUJU_MODEL_UUID'),
-                }
+                },
+                'subnet': SUBNET
```

Comment:
what about reading it from an environment value with default subnet to 10.0.0.0/16?